spec/javascripts/app/views/publisher_view_spec.js

```diff
@@ -542,7 +542,7 @@ describe("app.views.Publisher", function() {
         var upload_view = this.view.viewUploader;
         this.uploader = {
           onProgress: _.bind(upload_view.progressHandler, upload_view),
-          onSubmit:   _.bind(upload_view.submitHandler, upload_view),
+          onUpload:   _.bind(upload_view.uploadHandler, upload_view),
```

Comment:
- Extra space before value for key 'onUpload'.
- Identifier 'upload_view' is not in camel case.